pkg/wpa/passphrase/passphrase_test.go

```diff
@@ -0,0 +1,103 @@
+package passphrase
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+)
+
+type RunTestCase struct {
+	name  string
+	essid string
+	pass  string
+	out   []byte
+	err   error
+}
+
+var (
+	essidStub     = "stub"
+	shortPass     = "aaaaaaa"                                                          // 7 chars
+	longPass      = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa" // 64 chars
+	validPass     = "aaaaaaaaaaaaaaaa"                                                 // 16 chars
+	correctOutput = []byte(
```

Comment:
why not corectOutput = `whatevver` and then you can say
string(out) == correctOutput
below
https://play.golang.org/p/CnzYN3bnjIa

Reply:
I do this to ensure that the output from the Run function will be of the correct type

Reply:
that's not needed. The type is defined by the run function, right? You don't need to ensure that.

Reply:
That said it's not a big deal!